src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -271,6 +273,27 @@ $row-data-header-width: 200px;
             }
         }
     }
+    .filter-container {
+        font-family: Mada;
+        @media screen and (max-width: $screen-xs-min) {
+            font-size: 0.8rem;
+        }
+    }
+    .results-count {
+        font-family: Mada;
+        font-size: 1rem;
+        margin: 20px 0;
+        @media screen and (max-width: $screen-xs-min) {
+            font-size: 0.8rem;
+        }
+        .bold-total {
+            font-size: 1.1rem;
+            @media screen and (max-width: $screen-xs-min) {
+                font-size: 0.9rem;
+            }
+        }
+        text-align: left;
+    }
```

Comment:
Shouldn’t cause general problems, but just so you’re aware we’re now mixing desktop first and mobile first styles on encoded. Hopefully it won’t cause too many problems down the line. If we redo all the CSS going forward I would choose desktop first, but mobile first was the style of the day and that’s what bootstrap used which is why they used constants with “min” at the end.

Reply:
As I was fixing some minor CSS things this morning I was starting to think that the CSS for the body map is generally pretty messy and should be cleaned up. The trouble is that there is CSS for the body map as displayed on the summary page and different CSS for the body map as a thumbnail, and it should probably all go in a new file (instead of _summary.scss and _matrix.scss) and be clearly identifiable for each implementation. I think if we put the thumbnail on any other page this should be cleaned up for sure. I could also do it as part of this ticket.

Reply:
I think the body map is big and isolated enough concept, a new scss file would be a good idea.